sdk/script/types.ts

```diff
@@ -17,4 +17,9 @@ export interface Session {
     machineName: string;
 }
 
+export interface IPackageFile {
```

Comment:
This seems to only be used internally within the SDK (after your changes the CLI no longer needs it). I'm wondering if we could just move this interface into `management-sdk.ts` and rename it to something like `PackageFile`? We do have a few legacy naming inconsistencies as you can see :)